pkg/testgridanalysis/testidentification/ocp_variants.go

```diff
@@ -12,6 +12,7 @@ var (
 	// variant regexes
 	awsRegex     = regexp.MustCompile(`(?i)-aws`)
 	azureRegex   = regexp.MustCompile(`(?i)-azure`)
+	canaryRegex  = regexp.MustCompile(`(?i)-canary`)
```

Comment:
if you want this i won't say no, but there is only a single "canary" job(per release), so it's not really aggregating anything.  You can just look at the canary job directly if you care about it its health.

Reply:
Yea, I wasn't sure if it was useful -- if it's a variant then it shows up under it's own card  under "Variant status."  Maybe the better thing to do is show a warning if it gets below some high percentage.